readme.adoc

```diff
@@ -154,16 +156,9 @@ export PATH=$ISTIO_HOME/bin:$PATH
 [source,bash]
 ----
 oc login $(minishift ip):8443 -u admin -p admin
-oc adm policy add-scc-to-user anyuid -z istio-ingress-service-account -n istio-system
-oc adm policy add-scc-to-user anyuid -z default -n istio-system
-oc adm policy add-scc-to-user anyuid -z grafana -n istio-system
-oc adm policy add-scc-to-user anyuid -z prometheus -n istio-system
```

Comment:
Do we not need any of these permissions anymore?

Reply:
No, because now the `anyuid` addon is installed. With just these permissions I was not able to make it run istio 0.8.0